docker/publish-by-arch.sh

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+ARCH="x86"
+NEED_BUILD=false
+NEED_TAG=false
+NEED_PUBLISH=false
+NEED_MANIFEST=false
+
+SRC_POSTFIX=""
+DES_POSTFIX="x86"
+
+SHELL_FOLDER=$(cd "$(dirname "$0")";pwd)
+
+MVN_VERSION=`python ./get-project-version.py`
+
+buildImage() {
+  echo "Start building images"
+  if [ $ARCH = "aarch64" ]; then
+    sh build-arm-docker-images.sh
```

Comment:
I think we should prepare all package files first, like using `mvn clean install -DskipTests`

Reply:
```shell
mvn clean package -DskipTests -Pdocker
```

Reply:
`build-arm-docker-images.sh` only need the package files, and build the docker image manually, so `mvn clean install -DskipTests` is enough to use.

Reply:
OK. Got it.